### PR TITLE
[fix] GestureDetector: deep copies of events for multiswipes when rotated

### LINKED ISSUE #4724: Landscape: swiping & wrong place in book
* KOReader version: v2019.02-155-g8ecdf9d_2019-03-04/  
* Device: KA1

#### Issue
Maybe you are already aware of this.

Since the introduction of multiswipes, swiping (including single swipes) in landscape is completely f*cked up. The biggest problem with this is that you can't even open the menus with a swipe.
Luckily it´s still possible to open the menus by tapping.

Another thing is that you end up 1 or 2 pages back (after a tap). Reproduce like this:
Go from portrait to landscape. Tap once (forward or backward).

### PR DESCRIPTION
**Pull request by Frenzie**
Fixes #4724.

### CODE REVIEW

**Comment by Frenzie** [comment]
There's also some rotation-related issue with #4666.

File: frontend/device/gesturedetector.lua [hunk 1]
```diff
@@ -556,16 +557,22 @@ function GestureDetector:handlePan(tev)
         end
 
         if msd_distance > self.MULTISWIPE_THRESHOLD then
+            local pan_ev_ms = pan_ev
```
Comment (poire-z) [inline frontend/device/gesturedetector.lua]: I just pondered for a few... thousands of milliiseconds :) with that `_ms` suffix - which looks too much like what we'd use to indicate milliseconds. Any other possible suffix? _msd _multiswipe?

Comment (Frenzie) [reply frontend/device/gesturedetector.lua]: I could also just call it _copy, although that might be a bit misleading when it's not a copy.

**Comment by Frenzie** [comment]
@poire-z Any further comments? :-)

**Comment by poire-z** [comment]
None. Not as deep as you in those low-level thingies, and can't test right now, but if it works for you :)

**Comment by Frenzie** [comment]
PS There was no such thing as a rotated panning issue. That was just #4729.

**Pull request merged**